pkg/config/registry.go

```diff
@@ -31,10 +30,9 @@ func ReadCredentials() (username, password string, err error) {
 
 // ReadCredentialsFromSecret reads from Kubernetes secret registry-credentials.
 // Returns the username and password, or error.
-func ReadCredentialsFromSecret(ctx context.Context, client client.Client) (username, password string, err error) {
+func ReadCredentialsFromSecret(ctx context.Context, client cluster.Client) (username, password string, err error) {
```

Comment:
what was this change for?

Reply:
This was reverted.

It was for allowing this method to be compatible with `client.Client` (for the controller) and `kubernetes.Client` (for the CLI) that was being passed to the HelmClientFactory. However, we no longer have this problem after adding the difference `HelmClientFactory` implementation for the CLI flow.